data-archive/archive-jobs/mosip_credential/README.md

```diff
@@ -1,38 +1,53 @@
 # Archival Jobs
-The Jobs will archive the data from the source database to the destination database
-in mosip_credential db.
+The Jobs will archive the data from the **credential_transactions** table in mosip_credential db to the table in archive database.
 
 ## Pre-requisite steps:
 
 1. If we don't have mosip_archive DB than go ahead and continue with the [db_scripts](../../../data-archive/db_scripts)
 2. Once we have the mosip_archive DB ready we need to continue with the archive_jobs.
 3. Install python3.9 virtual env.
 4. Switch to virtual env.
-5. Install required psycopg dependency. `sudo yum install python3-psycopg
+5. Install required psycopg dependency. `sudo yum install python3-psycopg.
 
```

Comment:
Lets say "Below are the 2 different ways of running the the archival script.
1. Archive jobs based on status code (Recommended)
2. Archive jobs based on no of days after the records were created (Old script)

Reply:
DOne.